source/developers/form-sources/form-source-mediaconvert-transcode.rst

```diff
@@ -0,0 +1,38 @@
+:is-up-to-date: True
+
+:orphan:
+
+.. document does not appear in any toctree, this file is referenced
+   use :orphan: File-wide metadata option to get rid of WARNING: document isn't included in any toctree for now
+
+.. index:: Data Sources; Video Upload then Transcode Video from S3 Repository using MediaConvert
+
+.. _form-source-mediaconvert-transcode:
+
+==================================================================
+Video Upload then Transcode from S3 using MediaConvert Data Source
+==================================================================
+
+.. image:: /_static/images/form-sources/form-source-s3-transcode.png
+    :width: 50%
+    :alt: Source Control Video Transcoding from S3 Repository
+    :align: center
+
+-------------
+Configuration
+-------------
+
+.. image:: /_static/images/form-sources/form-source-s3-transcode-conf.png
+    :width: 50%
+    :alt: Source Control Video Transcoding from S3 Repository Properties
+    :align: center
+
+.. include:: /includes/form-sources/form-source-field-basics.rst
+
++------------------------+-----------------------------------------------------------------------------+
+|| Description/Purpose   || Data source to upload videos to WebDav repository                          |
```

Comment:
I think we need to replace `WebDav` with `AWS MediaConvert` or something like that